pennylane/io.py

```diff
@@ -146,13 +156,99 @@ def circuit_loaded_qiskit_circuit():
         return load(quantum_circuit, format="qiskit", measurements=measurements)
     except ValueError as e:
         if e.args[0].split(".")[0] == "Converter does not exist":
-            raise RuntimeError(
-                "Conversion from Qiskit requires the PennyLane-Qiskit plugin. "
-                "You can install the plugin by running: pip install pennylane-qiskit. "
-                "You may need to restart your kernel or environment after installation. "
-                "If you have any difficulties, you can reach out on the PennyLane forum at "
-                "https://discuss.pennylane.ai/c/pennylane-plugins/pennylane-qiskit/"
-            ) from e
+            raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
+        raise e
+
+
+def from_qiskit_op(qiskit_op, params=None, wires=None):
+    """Loads Qiskit SparsePauliOp objects by using the converter in the PennyLane-Qiskit plugin.
```

Comment:
> Can we provide a link to `SparsePauliOp` in the Qiskit docs.

Sure, can do! I didn't include it before to be consistent with the `from_qiskit()` docstring.

Reply:
> Also, I think we don't need to mention that this comes from the PL-Qiskit plugin.

Actually, this is the documentation convention for all of the `from_XXX()` convenience functions in this file so I would prefer to keep things consistent unless we have a good reason not to.

Reply:
> Sure, can do! I didn't include it before to be consistent with the from_qiskit() docstring.

Ah, we can break convention with `from_qiskit()` as we'll be revisiting that with the docs improvements next week.

Reply:
> Actually, this is the documentation convention for all of the from_XXX() convenience functions in this file so I would prefer to keep things consistent unless we have a good reason not to.

Also don't mind breaking convention here. My motivation is that if you're a user who sees `from_qiskit_op`, you probably don't care too much that the functionality lives in the PL-Qiskit plugin. Seems more like an implementation detail.

Reply:
Also, will we be able to backport these docs updates to the docstrings in the PL Qiskit plugin?

Reply:
> probably don't care too much that the functionality lives in the PL-Qiskit plugin. Seems more like an implementation detail.

@trbromley On one hand, I agree, but on the other hand, this informs the user that the PL-Qiskit plugin is required.

Reply:
It's a good point. Let's merge this in for now. Next week we could think about instead having it as a separate note rather than in the first line of the docstring. The note can give a few more details, like installation instructions.